### ISSUE TITLE
Unable to import Keras model - "Unexpected end-of-input: was expecting closing quote for a string value"

### ISSUE THREAD

**Issue by s-gbz** [deeplearning4j/deeplearning4j#7089]
#### Issue Description
I created+trained a simple Keras model, which can be found 
- [here](https://gist.github.com/s-gbz/daec03d605dcc55d0b7d03eb0c959509) and tried importing it in DL4J for convenient use with Spring Boot. 
- The Java code can be found [here](https://gist.github.com/s-gbz/eb24f4dbc25efcdfa15603e278dfe44b).
- The saved [.h5 file](https://github.com/adessoAG/facenality-model/blob/master/facenality/facenality-model-neutral-corrected-new-save.h5).

When trying to import and use the model, a problem concerning the .h5 file pops up.

`o.d.nn.modelimport.keras.Hdf5Archive     : Unexpected end-of-input: was expecting closing quote for a string value
 at [Source: {"config": [{"config": {"bias_initializer": {"config": {}, "class_name": "Zeros"}, "activity_regularizer": null, "kernel_size": [3, 3], "filters": 32, "trainable": true, "use_bias": true, "dilation_rate": [1, 1], "kernel_regularizer": null, "bias_constraint": null, "padding": "valid", "batch_input_shape": [null, 224, 224, 3], "kernel_initializer": {"config": {"mode": "fan_avg", "distribution": "uniform", "scale": 1.0, "seed": null}, "class_name": "VarianceScaling"}, "activation": "relu", "name": "conv2d_1", "dtype": "float32", "bias_regularizer": null, "data_format": "channels_last", "strides": [1, 1], "kernel_constraint": null}, "class_name": "Conv2D"}, {"config": {"name": "max_pooling2d_1", "trainable": true, "strides": [2, 2], "data_format": "channels_last", "pool_size": [2, 2], "padding": "valid"}, "class_name": "MaxPooling2D"}, {"config": {"trainable": true, "data_format": "channels_last", "name": "flatten_1"}, "class_name": "Flatten"}, {"config": {"bias_initializer": {"config": {}, "class_name": "Zeros"}, "activity_regularizer": null, "units": 16, "trainable": true, "use_bias": true, "kernel_regularizer": null, "bias_constraint": null, "batch_input_shape": [null, 224], "kernel_initializer": {"config": {"maxval": 0.05, "seed": null, "minval": -0.05}, "class_name": "RandomUniform"}, "activation": "relu", "name": "dense_1", "dtype": "float32", "bias_regularizer": null, "kernel_constraint": null}, "class_name": "Dense"}, {"config": {"bias_initializer": {"config": {}, "class_name": "Zeros"}, "activity_regularizer": null, "units": 16, "trainable": true, "use_bias": true, "kernel_regularizer": null, "bias_constraint": null, "batch_input_shape": [null, 224], "kernel_initializer": {"config": {"maxval": 0.05, "seed": null, "minval": -0.05}, "class_name": "RandomUniform"}, "activation": "relu", "name": "dense_2", "dtype": "float32", "bias_regularizer": null, "kernel_constraint": null}, "class_name": "Dense"}, {"config": {"bias_initializer": {"config": {}, "class_name": "; line: 1, column: 4001]`

`
2019-01-29 16:48:20.948  INFO 14504 --- [nio-8080-exec-1] org.nd4j.linalg.factory.Nd4jBackend      : Loaded [CpuBackend] backend
2019-01-29 16:48:21.412  INFO 14504 --- [nio-8080-exec-1] org.nd4j.nativeblas.NativeOpsHolder      : Number of threads used for NativeOps: 4
2019-01-29 16:48:21.657  INFO 14504 --- [nio-8080-exec-1] org.nd4j.nativeblas.Nd4jBlas             : Number of threads used for BLAS: 4
2019-01-29 16:48:21.665  INFO 14504 --- [nio-8080-exec-1] o.n.l.a.o.e.DefaultOpExecutioner         : Backend used: [CPU]; OS: [Windows 10]
2019-01-29 16:48:21.666  INFO 14504 --- [nio-8080-exec-1] o.n.l.a.o.e.DefaultOpExecutioner         : Cores: [8]; Memory: [7,1GB];
2019-01-29 16:48:21.667  INFO 14504 --- [nio-8080-exec-1] o.n.l.a.o.e.DefaultOpExecutioner         : Blas vendor: [MKL]
{config={lr=1.0, epsilon=1.0E-7, rho=0.95, decay=0.0}, class_name=Adadelta}
2019-01-29 16:48:22.010  INFO 14504 --- [nio-8080-exec-1] o.d.nn.multilayer.MultiLayerNetwork      : Starting MultiLayerNetwork with WorkspaceModes set to [training: ENABLED; inference: ENABLED], cacheMode set to [NONE]
2019-01-29 16:48:22.134 ERROR 14504 --- [nio-8080-exec-1] o.a.c.c.C.[.[.[/].[dispatcherServlet]    : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NullPointerException: input is marked @NonNull but is null] with root cause`

#### Version Information
- Windows 10, 64 Bit
- Gradle 4.8.1

- 'deeplearning4j-core', version: '1.0.0-beta2' / same results if using 1.0.0-beta3
- 'deeplearning4j-modelimport', version: '1.0.0-beta2' / same results if using 1.0.0-beta3
- 'nd4j-native-platform', version: '1.0.0-beta2' / same results if using 1.0.0-beta3

Using version '0.9.1' leads to `"org.deeplearning4j.nn.modelimport.keras.UnsupportedKerasConfigurationException: 
Unsupported keras layer type Conv2D."`

#### Contributing
I tried checking the .h5 file, but couldn't identify if it was corrupted. Please advice if I can help you out somehow.
PS: Sorry the code quality is rather sketchy. Need to get this done as a prove of concept :)

**Comment by saudet** [comment]
Please try again with 1.0.0-SNAPSHOT:
https://deeplearning4j.org/docs/latest/deeplearning4j-config-snapshots

**Comment by maxpumperla** [comment]
@s-gbz I checked your file and you should be fine with anything that's > 0.9.2. The Hdf5Archive warning is really just a warning, your model should load fine.
